cmd/odf/restore/crds.go

```diff
@@ -0,0 +1,27 @@
+package restore
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
+	"github.com/spf13/cobra"
+)
+
+// crdsCmd represents the mons command
+var crdsCmd = &cobra.Command{
+	Use:                "crds",
+	Short:              "Restores a CR that was accidentally deleted and is still in terminating state.",
+	DisableFlagParsing: true,
+	Args:               cobra.RangeArgs(1, 2),
+	Example:            "odf restore crds <CRD> [CRNAME]",
```

Comment:
What about the command `deleted`? IMO it's important to be clear in the command name that the CRs to be restored are in the deleted state.
```suggestion
	Example:            "odf restore deleted <CRD> [CRNAME]",
```

Reply:
agreed, updated the command.